public/app/features/alerting/unified/components/rules/RuleDetailsActionButtons.tsx

```diff
@@ -142,9 +142,12 @@ export const RuleDetailsActionButtons: FC<Props> = ({ rule, rulesSource }) => {
     const sourceName = getRulesSourceName(rulesSource);
     const identifier = ruleId.fromRulerRule(sourceName, namespace.name, group.name, rulerRule);
 
-    const editURL = urlUtil.renderUrl(`/alerting/${encodeURIComponent(ruleId.stringifyIdentifier(identifier))}/edit`, {
-      returnTo,
-    });
+    const editURL = urlUtil.renderUrl(
+      `${config.appSubUrl ?? ''}/alerting/${encodeURIComponent(ruleId.stringifyIdentifier(identifier))}/edit`,
```

Comment:
Is the nullish coalescing needed here? I don't see it used elsewhere. What does `config.appSubUrl` return if it's not configured?

Reply:
Ops. missed that